### PR TITLE
cp edits - servers, second batch

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-0385fa1599/how-to/). :bow:

File: content/cloud-servers/cloud-backup-basic-troubleshooting-on-linux-servers.md [hunk 1]
```diff
@@ -29,44 +29,47 @@ you create one.
 Before you restart the DriveClient service, check [status.rackspace.com](https://status.rackspace.com)
 to ensure that there are no open issues for Cloud Backup that could be impacting your server instance.
 In addition, check the current support tickets to ensure that there has not been an incident
-that is causing the lack of server accessibility. 
+that is causing the lack of server accessibility.
 
-To check for open support tickets, log in to the [Cloud Control Panel](https://mycloud.rackspace.com/)
+To check for open support tickets, log in to the [Cloud Control Panel](https://login.rackspace.com/)
```
[the remark this answers is not in the document]
Comment (catlook) [reply content/cloud-servers/cloud-backup-basic-troubleshooting-on-linux-servers.md]: nah...you can do it from the main login portal before choosing a product.

File: content/cloud-servers/migrate-from-parse-to-rackspace.md [hunk 2]
```diff
@@ -5,36 +5,45 @@ title: Migrate From Parse To Rackspace
...
 
 ### Deploy the Parse stack
 
-1. Log in to the [Cloud Control Panel](https://mycloud.rackspace.com).
+1. Log in to the [Cloud Control Panel](https://login.rackspace.com).
 
-2. In the top navigation bar, click **Orchestration** > **Stack Templates**.
+2. In the top navigation bar, click **Select a Product > Rackspace Cloud**.
 
-3. In the All Stacks list, scroll to **Parse** and click **Create Stack**.
```
[the remark this answers is not in the document]
Comment (catlook) [reply content/cloud-servers/migrate-from-parse-to-rackspace.md]: it's there

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f52d02568b/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-022e5f8880/how-to/). :bow:

**Pull request merged**